payload/usr/local/sal/checkin_modules/machine_checkin.py

```diff
@@ -111,17 +112,13 @@ def get_model_code(serial):
 def query_apple_support(serial):
     model_code = get_model_code(serial)
     tree = ElementTree.ElementTree()
+    session = macsesh.Session()
```

Comment:
macsesh needs to be imported into this file before it can be called. :)

Reply:
Good call. Fixing.